File: android studio/Web Servisi/HotelWIEN_API/ModelVM/RezervacijeResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace HotelWIEN_API.ModelVM
{
    public class RezervacijeResultVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int TuristID { get; internal set; }
            public string NazivSobe { get; internal set; }
            public DateTime DatumZavrsetka { get; internal set; }
            public int RezervacijaID { get; internal set; }
            public int SobaID { get; internal set; }
            public string Cijena { get; internal set; }
            public DateTime DatumPocetka { get; internal set; }
        }
    }
}

File: android studio/app/src/main/java/android/fit/ba/hotelwien/data/RezervacijeResultVM.java
package android.fit.ba.hotelwien.data;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class RezervacijeResultVM implements Serializable {

    public static class Row implements Serializable {
        public int TuristID ;
        public String NazivSobe;
        public Date DatumZavrsetka;
        public int RezervacijaID ;
        public int SobaID ;
        public String Cijena;
        public Date DatumPocetka ;
    }
    public List<Row> rows;

}

